### PR TITLE
🕷️  Default Values for Run time Config URL + Data Center

### PR DESCRIPTION
**Pull request by demariadaniel**
Protect against empty values for Config URLs (source of bug in Dev)

### CODE REVIEW

File: src/app/api/config/config.ts [hunk 1]
```diff
@@ -66,7 +66,7 @@ export const getAppConfig = (serverEnv: any): AppConfig => {
 			serverEnv.NEXT_PUBLIC_EGO_PUBLIC_KEY || process.env.NEXT_PUBLIC_EGO_PUBLIC_KEY || '',
 		UI_VERSION: packageJSON.version,
 		REGION: serverEnv.NEXT_PUBLIC_REGION || process.env.NEXT_PUBLIC_REGION || '',
-		DATA_CENTER: serverEnv.NEXT_PUBLIC_DATA_CENTER || process.env.NEXT_PUBLIC_DATA_CENTER || '',
+		DATA_CENTER: serverEnv.NEXT_PUBLIC_DATA_CENTER || process.env.NEXT_PUBLIC_DATA_CENTER || 'DC1',
```
Comment (joneubank) [inline src/app/api/config/config.ts:69]: What is `DC1` ?

**Review by joneubank — APPROVED** [review]
Thanks for putting this in place... the values you provided look like placeholders to make dev work, not actual values we want to default to.

It may be time to protect our config files correctly, where we throw an error on startup if required env variables are not provided.

File: src/app/api/config/config.ts (hunk 1 above)
Comment (demariadaniel) [reply src/app/api/config/config.ts:69]: DC1 is the only Data Center value we have at the moment

**Comment by demariadaniel** [comment]
> Thanks for putting this in place... the values you provided look like placeholders to make dev work, not actual values we want to default to.
> 
> It may be time to protect our config files correctly, where we throw an error on startup if required env variables are not provided.

@joneubank good point, I guess this is useful in the interim (since we are still testing and working on Dev/QA) but is worthy of some dedicated follow up

**Pull request merged**